registration-processor/workflow-engine/registration-processor-workflow-manager-service/src/main/resources/bootstrap.properties

```diff
@@ -0,0 +1,25 @@
+spring.cloud.config.uri=localhost
+spring.cloud.config.label=master
+spring.application.name=registration-processor,application
+spring.profiles.active=dev
+spring.cloud.config.name=registration-processor
+management.endpoints.web.exposure.include=refresh
+registration.processor.zone=secure
+vertx.cluster.configuration=${spring.cloud.config.uri}/*/${spring.profiles.active}/${spring.cloud.config.label}/hazelcast_${registration.processor.zone}.xml
+#management.security.enabled=false
+#disabling health check so that client doesnt try to load properties from sprint config server every
+# 5 minutes (should not be done in production)
+health.config.enabled=false
+eventbus.port=5754
+server.port=8026
+server.servlet.path=/registrationprocessor/v1/workflowmanager
+mosip.regproc.workflow.internal.action.server.port=8024

```

Comment:
Can we rename the properties that starts with mosip.regproc.workflow to mosip.regproc.workflow-manager.* ?

Reply:
Fixed